src/components/views/room_settings/RoomPublishSetting.tsx

```diff
@@ -41,6 +43,13 @@ export default class RoomPublishSetting extends React.PureComponent<IProps, ISta
         };
     }
 
+    private showError(): void {
+        Modal.createDialog(ErrorDialog, {
+            title: _t("room_settings|general|error_publishing"),
+            description: _t("room_settings|general|error_publishing_detail"),
+        });
+    }
+    
```

Comment:
I'd normally be the first person to advocate for splitting things out into a separate function, but in this case I think this is probably unnecessary.

Reply:
So no change, right?

Reply:
Well, I was suggesting just putting the code inline rather than in a separate function. Not a blocker though.

Reply:
okay, i thought this might be important considering the modular nature of code (more errors can be handled this way)

Reply:
Arguably, although in order to do this you'd really either need to have specific functions per error (eg. name it `showPermissionError`) or pass the error object to the function so it can decide which error to display.